src/operators/gdalsource.md

```diff
@@ -0,0 +1,32 @@
+# GdalSource
+
+The `GdalSource` is a _source operator_ that reads raster data using GDAL.
+The counter part for vector data is the `OgrSource`.
+
+## Parameters
+
+| Parameter | Type        | Description                        | Example Value | Default Value |
+| --------- | ----------- | ---------------------------------- | ------------- | ------------- |
+| `dataset` | `DatasetId` | The id of the dataset to be loaded | <code>"{<br>&nbsp;&nbsp;"type": "internal",<br>&nbsp;&nbsp;"datasetId": "a626c880-1c41-489b-9e19-9596d129859c"<br>}</code> | |
```

Comment:
Maybe put a `<pre>` around to get a nicer output of the example value.

Reply:
79b36d69847bd722e348825fe0903e7cb05ee27d